test/2_test_fullpath_with_permit.js

```diff
@@ -1793,6 +1812,25 @@ contract('Cashier and VoucherKernel', async (addresses) => {
           );
         });
 
+        it('[NEGATIVE] Should not create order from a wrong payment type', async () => {
```

Comment:
Whats the difference between these tests? they all look exactly the same to me unless i am missing something?

Reply:
Maybe you should have a look at the Builder